sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/DatabaseAccount.java

```diff
@@ -285,4 +285,9 @@ public String toJson() {
         this.populatePropertyBag();
         return super.toJson();
     }
+
+    @Override
+    public Object get(String propertyName) {
+        return super.get(propertyName);
```

Comment:
why do we need these overridden methods?
the implementation is the same as parent class.

Reply:
discussed offline.